prod_devel/domd.xml

```diff
@@ -7,13 +7,13 @@
 
     <default remote="github" sync-j="10" sync-c="true" />
 
-    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
-    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="rocko" revision="b704c689b67639214b9568a3d62e82df27e9434f" />
-    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
-    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
-    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
-    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
-    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="thud-dev" revision="2a535519e1808bb3a43e88a6c5dcc0556fc7101d" />
```

Comment:
why dev branch? there is a tag: thud-Yocto-v3.15.0?

Reply:
actually, this is tagged commit: https://github.com/renesas-rcar/meta-renesas/commits/thud-Yocto-v3.15.0